egs/swbd/s5c/local/nnet3/tuning/run_tdnn_c.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# c is as a, but with 1 more extra layer which gives 0.5-0.6% absolute 
+# improvement.
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+stage=0
+affix=
+train_stage=-10
+has_fisher=true
+speed_perturb=true
+common_egs_dir=
+reporting_email=
+remove_egs=true
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+suffix=
+if [ "$speed_perturb" == "true" ]; then
+  suffix=_sp
+fi
+dir=exp/nnet3/tdnn_a
+dir=$dir${affix:+_$affix}
+dir=${dir}$suffix
+train_set=train_nodup$suffix
+ali_dir=exp/tri4_ali_nodup$suffix
+
+local/nnet3/run_ivector_common.sh --stage $stage \
+	--speed-perturb $speed_perturb || exit 1;
+
+if [ $stage -le 9 ]; then
+  echo "$0: creating neural net configs";
+
+  # create the config files for nnet initialization
+  steps/nnet3/tdnn/make_configs.py  \
+    --feat-dir data/${train_set}_hires \
+    --ivector-dir exp/nnet3/ivectors_${train_set} \
+    --ali-dir $ali_dir \
+    --relu-dim 1024 \
+    --splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -3,3 -7,2 0"  \
```

Comment:
Hm.
I'm not too happy about checking in non-xconfig-based recipes at this point.
Is there any chance you could find the time to convert it to use xconfigs?  
It looks like we haven't checked in non-chain scripts using xconfigs yet, so this will be a good first step.
The non-chain system's xconfig is basically a simplification of the config for the chain-models... you could start with a chain TDNN's xconfig and modify it, while of course using the correct splice indexes for the TDNN.
You could do that as a separate experiment number in this sequence of experiments, so that if there are any changes arising from moving to xconfigs, we can see them clearly.

Basically I think you have to take a chain tdnn system using xconfigs,
remove the whole xent branch of the output, and change the output layer to be something like:

  output-layer name=output input=tdnn7 dim=$num_targets max-change=1.5                                       

(i.e. removing a bunch of options).
Check whether the max-change value is what we were using before by comparing with the configs of the regular nnet3 baseline.
You may not need the final "0" for the splice-indexes, because I think that is for the output layer.
Compare the config's structure (foo/bar/xconfigs/init.config, I think) with the structure from one of the later progress logs in your baseline nnet3 experiment: it should be about the same.